dockerfiles/nginx-tls/docker-entrypoint.sh

```diff
@@ -5,6 +5,10 @@ location_blocks="${LOCATION_BLOCKS:?LOCATION_BLOCKS not set}"
 location_blocks="$(echo "$location_blocks" | base64 -d)"
 export location_blocks
 
+log_format="${LOG_FORMAT:-I2xvZ19mb3JtYXQgaXMgbm90IGRlZmluZWQK}"
+log_format="$(echo "log_format" | base64 -d)"
```

Comment:
```suggestion
log_format="$(base64 -d <<<"$log_format")"
```

Reply:
pick either or, not both :)

Reply:
I would like keep the change consistent since line 5 is already using echo command and it is easier to understand.